vault/changelog.early-seed.md

```diff
@@ -2,29 +2,33 @@
 id: 3abd00eb-1c1e-4253-aaf5-dcbe20c21850
 title: Early Seed
 desc: ''
-updated: 1649439584096
+updated: 1650044828407
 created: 1604539200840
 published: true
 nav_exclude: true
 ---
 
 > This page documents the features, enhancements, and fixes in the latest early seed release. The early seed is a preview of the weekly release, so you'll see the next minor version when installing (ex. `0.78.0` instead of `0.77.1`). When dendron ships the general release, the currently installed extension will automatically be updated (but will still have same version).
 
-## 0.90.0
+## 0.91.0
 
-Dendron 0.90 has sprouted 🌱
+Dendron 0.91 has sprouted 🌱
 
-**Vault-Level URLs**: We're improving the experience with published vaults. Vaults can now be configured with their own URL. These URLs will be used when copying or exporting notes, replacing wikilinks with the correct URL for the vault. 
+**Note Title Configuaration**: This release introduces an configuration setting to allow notes to display their full hierarchy in their note title. For example, if this setting is enabled, a new note named `one.two.three` will be titled  `One Two Three`.
+
+**Go to Assets**: We've expanded functionality for file assets in Dendron. For wikilinks that link to non-note files, `Dendron: Go to Note` and `Go to Definition` now support opening these in a native way. 
```

Comment:
`now support opening these in a native way.` -> will now open them using your OS default app